app/controllers/charges_controller.rb

```diff
@@ -3,8 +3,8 @@ def new
   end
 
   def create
-    @total_amount = 50
-    @amount = (@total_amount*100)
+    @total_amount = Order.last.total
+    @amount = @total_amount.to_i*100
```

Comment:
yeah, well, it doesn't. order.total is also returning currency and such. Tried with just having @total_amount * 100 and it did not work.